packages/e2e-test-utils/src/customFonts.js

```diff
@@ -13,11 +13,54 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+function parseText(optionsText) {
+  return optionsText.map((option) => {
+    const font = option.split('\n');
+    return { name: font[0], url: font[1] };
+  });
+}
+
+/**
+ * Get custom font list
+ *
+ * @return {Promise<[]>}
+ */
+export const getFontList = async () => {
+  try {
+    const optionsText = await page.evaluate(() =>
+      Array.from(
+        document.querySelectorAll(
+          'div[role=listbox] [role=option] div:first-child'
+        ),
+        (element) => element.innerText
+      )
+    );
+    return parseText(optionsText);
+  } catch (e) {
+    return [];
+  }
+};
 
 /**
- * Internal dependencies
+ * Get custom font list
+ *
+ * @return {Promise<[]>}
  */
-import visitSettings from './visitSettings';
+export const getSelectedFontList = async () => {
```

Comment:
There's only ever one selected font, no? So why return an array? This is a bit misleading.

Reply:
Good point

Reply:
Updated here:
https://github.com/GoogleForCreators/web-stories-wp/pull/10918/files#diff-15102bcca0cb4bffec2cea43a88ada5c191041b40f7e784266ac7b5cd2161a1dR55